src/views/Home2/components/Hero.tsx

```diff
@@ -0,0 +1,136 @@
+import CompositeImage, { getSrcSet, CompositeImageProps } from './CompositeImage'
+
+const flyingAnim = () => keyframes`
+  from {
+    transform: translate(0,  0px);
+  }
+  50% {
+    transform: translate(-10px, -15px);
+  }
+  to {
+    transform: translate(0, 0px);
+  }  
+`
+
+const fading = () => keyframes`
+  from {
+    opacity: 0.9;
+  }
+  50% {
+    opacity: 0.1;
+  }
+  to {
+    opacity: 0.9;
+  }  
+`
+
+const BgWrapper = styled.div`
+  z-index: -1;
+  overflow: hidden;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  bottom: 0px;
+  left: 0px;
+`
+
+const InnerWrapper = styled.div`
+  position: absolute;
+  width: 100%;
+  bottom: -3px;
+`
+
+const BunnyWrapper = styled.div`
+  width: 100%;
+  animation: ${flyingAnim} 2.5s ease-in-out infinite;
+`
+
+const StarsWrapper = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  & :nth-child(2) {
+    animation: ${fading} 2s ease-in-out infinite;
+    animation-delay: 1s;
+  }
+
+  & :nth-child(3) {
+    animation: ${fading} 5s ease-in-out infinite;
+    animation-delay: 0.66s;
+  }
+
+  & :nth-child(4) {
+    animation: ${fading} 2.5s ease-in-out infinite;
+    animation-delay: 0.33s;
+  }
+`
+
+const Hero = () => {
+  const { t } = useTranslation()
+  const { account } = useWeb3React()
+  const { theme } = useTheme()
+  const imagePath = '/images/home/lunar-bunny/'
+  const imageSrc = 'bunny'
+
+  const starsImage: CompositeImageProps = {
+    path: '/images/home/lunar-bunny/',
+    attributes: [
+      { src: 'star-l', alt: '3D Star' },
+      { src: 'star-r', alt: '3D Star' },
+      { src: 'star-top-r', alt: '3D Star' },
+    ],
+  }
+
+  return (
+    <>
+      <BgWrapper>
+        <InnerWrapper>{theme.isDark ? <SlideSvgDark width="100%" /> : <SlideSvgLight width="100%" />}</InnerWrapper>
+      </BgWrapper>
+      <Flex
+        flexDirection={['column-reverse', null, null, 'row']}
+        alignItems={['flex-end', null, null, 'center']}
+        justifyContent="center"
+      >
+        <Flex flex="1" flexDirection="column">
+          <Heading scale="xxl" color="secondary" mb="24px">
+            {t('The moon is made of pancakes.')}
+          </Heading>
+          <Heading scale="md" mb="24px">
+            {t('Trade, earn, and win crypto on the most popular decentralized platform in the galaxy.')}
+          </Heading>
+          <Flex>
+            {!account && <ConnectWalletButton mr="8px" />}
+            <Link mr="16px" href="/swap">
+              <Button variant="secondary">{t('Trade Now')}</Button>
+            </Link>
+          </Flex>
+        </Flex>
+        <Flex
+          height={['192px', null, null, '100%']}
+          width={['192px', null, null, '100%']}
+          flex={[null, null, null, '1']}
+          mb={['24px', null, null, '0']}
+          position="relative"
+        >
+          <BunnyWrapper>
+            <img src={`${imagePath}${imageSrc}.png`} srcSet={getSrcSet(imagePath, imageSrc)} alt={t('Lunar bunny')} />
```

Comment:
Why not using the `Image` component ?

Reply:
The short answer here is that it's a bit of a pain to use that (if I remember correctly) relies on an explicit, numeric width/height? 

I think a less opinionated, raw `<img>` component that makes use of the lazyload functionality and optionly the `srcSet` attribute could be useful?